src/app/userDashboard/registerMatches/page.tsx

```diff
@@ -5,63 +5,54 @@ import { Navbar } from '../../../Components/Navbar/Navbar';
 import Image from 'next/image';
 import { sendRequest } from '@/services/auth/auth_All_Api';
 import { useSearchParams } from 'next/navigation';
-import InputCustomTag from '@/Components/InputCustomTag/InputCustomTag';
-//@ts-ignore
-import { Button, Input } from 'technogetic-iron-smart-ui';
-import { FormikHelpers, useFormik } from 'formik';
-import { SendInviteSchema } from '@/schemas/SignupSchemas';
+import { formatDate, formatTime } from '@/Components/CommonComponent/moment';
+import { toast } from 'react-toastify';
+import CountdownComponent from '../tournament/CountdownComponent'
 
 
 export interface RegMatch {
   gameName: string;
   gameType: string;
   mapType: string;
   version: string;
-  date: string;
-  time: string;
+  dateAndTime: string;
   lastSurvival: string;
   roomId: string;
+  roomUuid: string;
   password: string;
+  entryFee?: string;
+  mapImg: string;
 }
-interface FormValues {
-  email: string;
-}
-
 
 const regMatches = () => {
-  const [values, setValues] = useState<string[]>([]);
-  const [tags, setTags] = useState([]);
-
   const searchParams = useSearchParams();
   const matchID = searchParams.get('id');
   const [matchData, setMatchData] = useState<RegMatch>();
-  const [showRoomPwd, setRoomPwd] = useState<boolean>(false);
-  const getRegisterMatchWithId = async () => {
-    const token: any = localStorage.getItem('jwtToken');
-    const regMAtch = await sendRequest(`room/rooms/${matchID}`, {
-      method: 'GET',
-      headers: { Authorization: `Bearer ${token}` },
-    });
-    setMatchData(regMAtch.data.room);
-  };
+  const [visibleRooms, setVisibleRooms] = useState<string[]>([]);
 
-  const getRoomidPwd = () => {
-    const currentTime = new Date();
-    const dbTime = `${matchData?.date}T${matchData?.time}`;
-    const matchTime = new Date(dbTime);
-    const timeDifference = Number(matchTime) - Number(currentTime);
-    if (timeDifference <= 900000) {
-      setRoomPwd(true);
+  const getRegisterMatchWithId = async () => {
+    try {
+      const token: any = localStorage.getItem('jwtToken');
+      const regMAtch = await sendRequest(`room/rooms/${matchID}`, {
+        method: 'GET',
+        headers: { Authorization: `Bearer ${token}` },
+      });
+      setMatchData(regMAtch.data.room);
+      if (regMAtch.status === 200) {
+        const formatDateTime = ` ${formatDate({ date: matchData?.dateAndTime })} and ${formatTime({ time: matchData?.dateAndTime, format: 'LT' })}`;
+        setMatchData({ ...regMAtch.data.room, dateAndTime: formatDateTime })
+       CountdownComponent(matchData?.dateAndTime, matchData?.roomUuid, setVisibleRooms)
+      }
+    } catch (err) {
+      toast.error("not show data ")
     }
   };
 
   useEffect(() => {
     getRegisterMatchWithId();
+    
   }, []);
 
-  useEffect(() => {
-    getRoomidPwd();
-  }, [matchData]);
 
   return (
     <div className={styles.main_container}>
```

Comment:
If you have seen there uerTournament properly there is id given to this Div please check why and what is doing